webrender/src/tiling.rs

```diff
@@ -1431,7 +1431,7 @@ impl PrimitiveBatch {
         });
         let key = AlphaBatchKey::new(AlphaBatchKind::Composite,
                                      AlphaBatchKeyFlags::empty(),
-                                     BlendMode::Alpha,
+                                     BlendMode::PremultipliedAlpha,
```

Comment:
You're right, we should be using BlendMode::Alpha for the results of the MixBlendMode shader. I misunderstood how Gecko was handling this case.